app/javascript/components/concept-map/ConceptMap.css

```diff
@@ -69,32 +69,25 @@
                 }
 
                 & .display {
-                    @apply pt-12 pb-8 px-24 flex items-center justify-between;
+                    @apply py-12 px-24 flex items-center justify-between;
```

Comment:
Keep the same vertical padding top and bottom and use `display:none` instead of `visiblity:hidden` further down.

Reply:
This will throw off the alignment of the flexbox. also will make it appear as though the text is floating higher

Reply:
Nice catch. I've fixed it by adding `align-items:center` in the latest commit.

<img width="1016" alt="Screenshot 2020-10-30 at 22 52 33" src="https://user-images.githubusercontent.com/286476/97763164-b7a72080-1b02-11eb-92d0-3ae715392957.png">